integration_tests/suite/test_database.py

```diff
@@ -95,3 +102,39 @@ def test_subscription_option_unique(self):
                     raises(IntegrityError))
 
         self._Session.remove()
+
+    def test_subscription_pubusb_create(self):
+        service = SubscriptionService({'db_uri': self.db_uri})
+
+        tracker = {'uuid': None}
+
+        def on_subscription(subscription):
+            tracker['uuid'] = subscription.uuid
+
+        service.pubsub.subscribe('created', on_subscription)
+        service.create({'uuid': str(uuid.uuid4()), 'name': 'test',
+                        'owner_tenant_uuid': str(uuid.uuid4())})
+        assert_that(tracker, has_entries({
+            'uuid': is_(not_none())
```

Comment:
FYI we have a validator for [uuid](https://github.com/wazo-pbx/xivo-test-helpers/blob/master/xivo_test_helpers/hamcrest/uuid_.py) (but we do not think to use it everytime :sweat_smile: )